VinylUI/WinForms.fs

```diff
@@ -196,6 +196,11 @@ type BindPartExtensions =
     static member toModel (view: BindViewPart<Control, Nullable<'a>>, modelProperty: Expr<'a option>, ?sourceUpdateMode) =
         view.toModel(modelProperty, Option.ofNullable, Option.toNullable, ?sourceUpdateMode = sourceUpdateMode)
 
+    /// Create a two-way binding, automatically converting between string and string option.
```

Comment:
The "conversion" needs clarification. Here's my attempt: "converting between string and string option, where null and whitespace from the view becomes None on the model". Not terribly concise, but clear. Can you improve on this?